src/python/vsprojects/python_examples.pyproj

```diff
@@ -5,7 +5,7 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>8cbe34f8-5d46-4832-b615-db183812f81f</ProjectGuid>
     <ProjectHome>.</ProjectHome>
-    <StartupFile>..\examples\example16_coherence_score.py</StartupFile>
+    <StartupFile>..\examples\example_artm_model.py</StartupFile>
```

Comment:
I can't see example_artm_model.py in this pull request. Have you added it to git?